Files/MultilingualResources/Files.fr-FR.xlf

```diff
@@ -3256,21 +3256,22 @@
         </trans-unit>
         <trans-unit id="SettingsPrivacyPolicyMarkdownBlock.Text" translate="yes" xml:space="preserve">
           <source>
-*Personal Information Collection* 
+*Personal Information Collection*
 Files does not collect, store, share or publish any personal information.
 
 *Non-personal Information Collection*
 
 We use App Center to keep track of app usage, find bugs, and fix crashes. All information sent to App Center is anonymous and free of any user or contextual data.
   </source>
           <target state="translated">
-*Collecte d'informations personnelles* 
+*Collecte d'informations personnelles*
 Files ne collecte, ne stocke, ne partage ni ne publie aucune information personnelle.
 
 *Collecte d'informations non personnelles
 
 Nous utilisons l'App Center pour suivre l'utilisation de l'application, trouver des bogues et réparer les pannes. Toutes les informations envoyées à App Center sont anonymes et exemptes de toute donnée relative à l'utilisateur ou au contexte.
   </target>
+          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
```

Comment:
```suggestion
```

When you confirm that the new translation is accurate you can remove this line too. And after that I will approve this, since LGTM ;)